js/data_page.js

```diff
@@ -31,4 +40,170 @@ function copyToClipboard(elementId) {
   // Remove it from the body
   document.body.removeChild(aux);
 
-}
\ No newline at end of file
+}
+
+// Data Links and File Size JSON
+
+var data_file_sizes = {
+  "All Subsystems" : {
+    hour_size_CSV: "31.9 MB",
+    hour_size_JSON: "117.9 MB",
+    minute_size_CSV: "1.6 GB",
+    minute_size_JSON: "6.6 GB"
+  },
+  DHW : {
+    hour_size_CSV: "5.5 MB",
+    hour_size_JSON: "17.2 MB",
+    minute_size_CSV: "291 MB",
+    minute_size_JSON: "1 GB"
+  },
+  SHW: {
+    hour_size_CSV: "830.3 KB",
+    hour_size_JSON: "2.6 MB",
+    minute_size_CSV: "52 MB",
+    minute_size_JSON: "176.2 MB"
+  },
+  Elec: {
+    hour_size_CSV: "9 MB",
+    hour_size_JSON: "40.9 MB",
+    minute_size_CSV: "500.5 MB",
+    minute_size_JSON: "2.3 GB"
+  },
+  HVAC: {
+    hour_size_CSV:"1.3 MB",
+    hour_size_JSON:"3.8 MB",
+    minute_size_CSV: "64.3 MB",
+    minute_size_JSON: "230.8 MB"
+  },
+  IndEnv: {
+    hour_size_CSV:"5.1 MB",
+    hour_size_JSON:"12.6 MB",
+    minute_size_CSV: "269.5 MB",
+    minute_size_JSON: "737.9 MB"    
+  },
+  Load: {
+    hour_size_CSV:"4.6 MB",
+    hour_size_JSON:"25.6 MB",
+    minute_size_CSV: "241 MB",
+    minute_size_JSON: "1.4 GB"    
```

Comment:
Sorry if I've got this wrong, but isn't this 1.4 GB the one Pri referenced that needed to be converted to MB?

https://commercegov.slack.com/archives/nist-netzero/p1477600302000052